index.d.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace execall {
+	interface Match {
+		match: string;
+		sub: string[];
+		index: number;
+	}
+}
+
+/**
+Find multiple RegExp matches in a string.
+
+@param regexp - Regular expression to match against the `string`.
+@returns An array of matches.
```

Comment:
We don't need to mention the type here. Just:
```suggestion
@returns The matches.
```